x/staking/spec/07_params.md

```diff
@@ -1,14 +0,0 @@
-<!--
```

Comment:
I believe someone forgot to delete this file: there's already `07_events.md` in the spec, and a newer `08_params.md`